google-cloud-bigquery/lib/google/cloud/bigquery/project.rb

```diff
@@ -960,6 +960,34 @@ def schema
           s
         end
 
+        ##
+        # Creates a new encryption configuration instance.
```

Comment:
It might be good to explain how the configuration instance should be used, and that this method does not make a RPC.